mappings/1.7.2#13w47a.tinydiff

```diff
@@ -93,8 +93,8 @@ c	net/minecraft/unmapped/C_9685768
 		p	6		p_6	
 		p	7		p_7	
 c	net/minecraft/unmapped/C_6954368
-	f	I	f_1077993		f_1077993
-	f	I	f_5804353		f_5804353
+	f	I	f_1077993		damagedAtAge
```

Comment:
More of a nitpick, but elsewhere I've tried to format names like this as "<noun> of xyz" or similar rather than "<adjective> <noun>". So here `ageOfLastDamage` or `ageWhenLastDamaged` or something like that.